py/desitarget/mock/mockmaker.py

```diff
@@ -3254,24 +3259,23 @@ def read(self, mockfile=None, mockformat='CoLoRe', healpixels=None, nside=None,
         
         if self.mockformat == 'colore':
             self.default_mockfile = os.path.join(
-                os.getenv('DESI_ROOT'), 'mocks', 'lya_forest', 'london', 'v4.0', 'master.fits')
+                os.getenv('DESI_ROOT'), 'mocks', 'lya_forest', 'london', 'v4.2.0', 'master.fits')
+            #Should we set v9.0 as default?
             MockReader = ReadLyaCoLoRe()
```

Comment:
Yes, let's go ahead and set `v9.0` as the default. This change would only impact a power user who uses these classes "by hand", and having an updated default mockfile would be useful.